base/complex.jl

```diff
@@ -1033,3 +1033,12 @@ function complex(A::AbstractArray{T}) where T
     end
     convert(AbstractArray{typeof(complex(zero(T)))}, A)
 end
+
+# recursive arithmetic (fallback)
+
+for op ∈ (:div,:rem,:mod,:mod1,:fld,:fld1,:cld,:ldexp)
+    @eval $op(z::Complex,n) = Complex($op(real(z),n),$op(imag(z),n))
```

Comment:
As I explained in #37376, I think it would be better to use `n::Real` here if we are doing the computation by splitting up into real and imaginary parts.

Reply:
I want to see if there are any errors that result from this commit first, but I will change it later